src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -640,9 +653,104 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
                 frontend_logger.log("Message format not recognized", "error");
                 break;
             }
+            
+            frontend_logger.log(trainData, "info");
```

Comment:
We should have a meaningful log here. If its not really required, we should remove this line.

Reply:
This info log line has been deleted. Hence resolving this comment.